Makefile

```diff
@@ -40,6 +41,6 @@ build-docker:
 
 build-release:
 	@echo "== Release build =="
-	CGO_ENABLED=0 GOOS=$(TARGET) GOARCH=$(ARCH) go build -o $(BINARY_NAME)-$(TARGET)-$(ARCH)$(EXT) -v
+	CGO_ENABLED=0 GOOS=$(TARGET) GOARCH=$(ARCH) go build -ldflags "-X main.version=$(TRAVIS_TAG)" -o $(BINARY_NAME)-$(TARGET)-$(ARCH)$(EXT) -v
```

Comment:
Nitpicking but I would prefer having a variable for LDFLAGS, then we can add it to make run aswell.

Reply:
Sounds good!